src/centralstore/incremental/DataPublisher.h

```diff
@@ -0,0 +1,21 @@
+#include <arpa/inet.h>
```

Comment:
Add copyright header

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/122/commits/bdd734e88390169ddc9e44c74030d1c1b170d0ae

Reply:
Done